packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -70,22 +70,16 @@ export const generateTestDataForSlug = async (
   args.push("--slug", `"${slug}"`);
 
   const slugFields = new Set<string>();
-  let shouldAddDefaultSlugField = false;
   templateModuleCollection.forEach((templateModule) => {
     const slugField = templateModule?.config?.slugField;
     if (slugField) {
       slugFields.add(slugField);
     } else {
-      shouldAddDefaultSlugField = true;
+      slugFields.add("slug");
```

Comment:
We only want to add this default slug field if we encounter a dynamic template without slugField set. This will add it if we find a static template without slugField (which is every static template).